internal/venice-common/src/main/java/com/linkedin/venice/meta/Partition.java

```diff
@@ -26,19 +33,33 @@
 public class Partition {
   private static final Logger LOGGER = LogManager.getLogger(Partition.class);
   /**
-   * ID of partition. One of the number between [0 ~ total number of partition)
+   * ID of partition. One of the number between [0, total number of partition[
```

Comment:
maybe ] instead of [

Reply:
Hi Felix, closing this issue as Sushant also posted the same one. I might agree what he said - original syntax might be correct?